pkg/build/builder/docker.go

```diff
@@ -66,6 +66,7 @@ func (d *DockerBuilder) Build() error {
 	if err = d.addBuildParameters(buildDir); err != nil {
 		return err
 	}
+	glog.V(4).Infof("Starting Docker build from %s buildConfig", d.build.Name)
```

Comment:
I think this message qualifies as 'reporting progress' so I would suffix it with " ..."